content/rendering-with-batch/batch/monitor.md

```diff
@@ -1,11 +1,9 @@
 ---
-title: "Monitoring and results"
+title: "Monitoring AWS Batch"
```

Comment:
Why this change? This page talks about the outcome (result) of the rendering process as well as momitoring